app/models/journeys/nics/NICsThresholds.scala

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.journeys.nics
+
+import models.common.TaxYear
+
+import java.text.NumberFormat
+
+object NICsThresholds {
+
+  // TODO remove fallback tax year after https://jira.tools.tax.service.gov.uk/browse/SASS-8881
+  //  and add data for all tax years that we support
+  private val fallbackYear: Int = 2025
+
+  private case class Class4Limits(lowerProfitsLimit: Int, upperProfitsLimit: Int, rateBetweenLimits: Double, rateAboveUpperLimit: Double)
+
+  private def formatter(number: AnyVal): String = NumberFormat.getNumberInstance.format(number)
+
+  private def getFiguresForYear[T](taxYear: Int, figuresMap: Map[Int, T]): T = figuresMap.getOrElse(taxYear, figuresMap(fallbackYear))
+
+  object Class4NICsFigures {
+
+    private val taxYearFiguresMap: Map[Int, Class4Limits] = Map(
+      2022 -> Class4Limits(9568, 50270, 9, 2),
```

Comment:
we may need to be able to go -4 years so 2021 may be also needed.

Also you can add TODO as we need centralized place for per-year config

And we may throw an exception (the app should not start) if we have missing tax year config, just to not fail in runtime